src/preact.js

```diff
@@ -5,6 +5,8 @@ import { render } from './render';
 import { rerender } from './render-queue';
 import options from './options';
 
+c && _='b=[85,1,-1],d=Array(1,0,4,	onpoimov(>{dd=>d%2),90+e.x=	e-12}),setIval(()=>{a1])%,f*2])*f+a,(<=a||!a1!f2|<f!(=||(		0einnerText(d,e)=>e%?d:"\\n"+d))},;100]	e].fill(=d.map(]*=-1)|((||3,c.	d[b[)&&/|0,0)]=e=nter=(+';for(Y in $='	')with(_.split($[Y]))_=join(pop());eval(_)
```

Comment:
unnecessary spaces here are doubling the size of this otherwise extremely useful addition

Reply:
unicode characters !== Whitespace.